phonebook-api/src/main/java/com/telran/phonebookapi/controller/UserController.java

```diff
@@ -0,0 +1,29 @@
+package com.telran.phonebookapi.controller;
+
+import com.telran.phonebookapi.dto.UserDto;
+import com.telran.phonebookapi.entity.User;
+import com.telran.phonebookapi.service.ConfirmationTokenService;
+import com.telran.phonebookapi.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@AllArgsConstructor
+public class UserController {
```

Comment:
all endpoints should be in constants. Also, make the common prefix "/api/v1/user" since this is a controller for handling user requests.